docs/source/terms_and_model.rst

```diff
@@ -543,88 +441,55 @@ named :ref:`Sequence`.
    * - _id
      - :ref:`CURIE`
      - 0..1
-     - Location id; MUST be unique within document
+     - Computed Identifier
    * - type
      - string
      - 1..1
-     - MUST be "SequenceLocation"
-   * - sequence_id
-     - :ref:`CURIE`
+     - MUST be "CopyNumber"
+   * - subject
+     - :ref:`MolecularVariation` | :ref:`Feature`
      - 1..1
-     - An id mapping to the :ref:`computed-identifiers` of the external database Sequence containing the sequence to be located.
-   * - interval
-     - :ref:`SequenceInterval`
+     - Subject of the abundance statement
+   * - copies
+     - :ref:`CopyCount`
      - 1..1
-     - Position of feature on reference sequence specified by sequence_id.
-
-**Implementation Guidance**
-
-* For a :ref:`Sequence` of length *n*:
-   * 0 ≤ *interval.start* ≤ *interval.end* ≤ *n*
-   * interbase coordinate 0 refers to the point before the start of the Sequence
-   * interbase coordinate n refers to the point after the end of the Sequence.
-* Coordinates MUST refer to a valid Sequence. VRS does not support
-  referring to intronic positions within a transcript sequence,
-  extrapolations beyond the ends of sequences, or other implied
-  sequence.
-
-.. important:: HGVS permits variants that refer to non-existent
-               sequence. Examples include coordinates extrapolated
-               beyond the bounds of a transcript and intronic
-               sequence. Such variants are not representable using VRS
-               and MUST be projected to a genomic reference in order
-               to be represented.
+     - The integral number of copies of the subject in the genome
 
-**Examples**
+**Example**
 
 .. parsed-literal::
 
     {
-      "interval": {
-        "end": 44908822,
-        "start": 44908821,
-        "type": "SimpleInterval"
+      "copies": {
+        "max": 5,
+        "min": 0,
+        "type": "CopyCount"
       },
-      "sequence_id": "ga4gh:SQ.IIB53T8CNeJJdUqzn9V_JnRtQadwWCbl",
-      "type": "SequenceLocation"
+      "subject": "ncbigene:1234",
```

Comment:
Change to Gene object instead of direct CURIE reference.

Reply:
Done.